src/integration-test/java/uk/gov/hmcts/juror/api/moj/report/standard/PanelSummaryReportITest.java

```diff
@@ -0,0 +1,134 @@
+        return StandardReportResponse.builder()
+            .headings(new ReportHashMap<String, StandardReportResponse.DataTypeValue>()
+                .add("panel_summary", StandardReportResponse.DataTypeValue.builder()
+                    .displayName("Panel Summary")
+                    .dataType("Long")
+                    .value(1)
+                    .build())
+                .add("trial_number", StandardReportResponse.DataTypeValue.builder()
+                    .displayName("Trial Number")
+                    .dataType("String")
+                    .value("T100000001")
+                    .build())
+                .add("names", StandardReportResponse.DataTypeValue.builder()
+                    .displayName("Names")
+                    .dataType("String")
+                    .value("TEST DEFENDANT")
+                    .build())
+                .add("court_room", StandardReportResponse.DataTypeValue.builder()
+                    .displayName("Court Room")
+                    .dataType("String")
+                    .value("large room fits 100 people")
+                    .build())
+                .add("judge", StandardReportResponse.DataTypeValue.builder()
+                    .displayName("Judge")
+                    .dataType("String")
+                    .value("Test judge")
+                    .build())
+                .add("court_name", StandardReportResponse.DataTypeValue.builder()
+                    .displayName("Court Name")
+                    .dataType("String")
+                    .value("CHESTER (415)")
+                    .build()))
+            .tableData(
+                StandardReportResponse.TableData.<List<LinkedHashMap<String, Object>>>builder()
+                    .headings(List.of(
+                        StandardReportResponse.TableData.Heading.builder()
+                            .id("juror_number")
+                            .name("Juror Number")
+                            .dataType("String")
+                            .headings(null)
+                            .build(),
+                        StandardReportResponse.TableData.Heading.builder()
+                            .id("first_name")
+                            .name("First Name")
+                            .dataType("String")
+                            .headings(null)
+                            .build(),
+                        StandardReportResponse.TableData.Heading.builder()
+                            .id("last_name")
+                            .name("Last Name")
+                            .dataType("String")
+                            .headings(null)
+                            .build()))
+                    .data(List.of(
+                        new ReportLinkedMap<String, Object>()
+                            .add("juror_number", "415000001")
+                            .add("first_name", "FNAME1")
+                            .add("last_name", "LNAME1")))
+                    .build())
```

Comment:
Ideally we need at least three unique items in here so three jurors (Just to verify we can return multiple)